packages/node/src/indexer/BlockedQueue.ts

```diff
@@ -22,10 +22,31 @@ export class BlockedQueue<T> {
     this._queue.push(item);
   }
 
+  putAll(items: T[]): void {
+    if (this._queue.length >= this._maxSize) {
+      throw new Error('BlockedQueue exceed max size');
+    }
+    this._queue = this._queue.concat(items);
```

Comment:
```suggestion
       this._queue.push(...item);
```
It mutates the original object, not copy a new one